Resources/Prototypes/Entities/Objects/Power/antimatter_part.yml

```diff
@@ -2,7 +2,7 @@
   parent: BaseItem
   id: AMEPart
   name: AME part
-  description: A flatpack used for constructing an antimatter engine reactor.\nUse a multitool to unpack it.
+  description: A flatpack used for constructing an antimatter engine reactor. Use a multitool to unpack it.
```

Comment:
\n doesn't seem to work when placed in description field. I didn't change anything related to item descriptions in my original PR, so I'm not sure why it shows as a literal \n on examine

Reply:
It's probably a mistaken addin from byond.